node/src/components/small_network.rs

```diff
@@ -675,36 +670,6 @@ where
             .ignore()
     }
 
-    /// Sweeps across connection symmetry, enforcing symmetrical connections.
-    fn enforce_symmetric_connections(&mut self, now: Instant) -> Effects<Event<P>> {
```

Comment:
This function is currently being put "back into use" as part of #1844, similar to `SweepSymmetries`. How do we want to tackle this?

Reply:
(We will have a merge conflict, depending on which is merged first).

Reply:
I noticed that too, but I think it's such a trivial conflict, it's better to just remove it here and re-add it in your PR.

Reply:
Ha, easy for you to say, but fair enough I guess :)